launcher/preload.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+copy_launcher() {
+  cp launcher/launcher /usr/share/oem/cc_container_launcher
+}
+
+setup_launcher_systemd_unit() {
+  cp launcher/container-runner.service /usr/share/oem/container-runner.service
+  # set attest service endpoint
+  sed -i 's/\${ATTEST_ENDPOINT}/'${ATTEST_ENDPOINT}'/g' /usr/share/oem/container-runner.service
```

Comment:
Can we specify this in a .conf file instead of `sed`ing it? https://www.freedesktop.org/software/systemd/man/systemd.service.html#Command%20lines

See more at https://www.freedesktop.org/software/systemd/man/systemd.unit.html (search for .conf files)

Reply:
I think it is possible, ATTEST_ENDPOINT is an env variable passed in when executing preload.sh, you mean we create a .conf file on the fly in this function?